src/oapi/vms.rs

```diff
@@ -383,6 +417,20 @@ impl VmSpecs {
         Some(self)
     }
 
+    fn parse_dedicated_instance_additional_prices(mut self, input: &Input) -> Option<VmSpecs> {
+        let dedicated_vm_additional =
+            input.catalog_entry("TinaOS-FCU", "DedicatedInstanceSurplus", "RunInstances")?;
+        self.factor_vm_additional_cost = match self.tenancy.as_str() {
+            "dedicated" => 1.0 + ((dedicated_vm_additional * 10000.0) / 100.0),
+            "default" => 1.0,
+            _ => {
+                warn!("error to calculate factor");
+                0.0
+            }
+        };
```

Comment:
- If its an error, you should use `error!`
- Also default to 1.0, that's better than having the whole vm price going to zero.
- Please provide a more contextual warning such as:
```
unknown_tenancy => {
    error!("vm additional costs for {} is not supported", unknown_tenancy);
    1.0
}
```